src/Avalonia.Controls/Button.cs

```diff
@@ -493,7 +477,7 @@ private static void IsCancelChanged(AvaloniaPropertyChangedEventArgs e)
         /// </summary>
         /// <param name="sender">The event sender.</param>
         /// <param name="e">The event args.</param>
-        private void CanExecuteChanged(object sender, EventArgs e)
+        public void CanExecuteChanged(object sender, EventArgs e)
```

Comment:
Why was it changed to public?
It was implicitly implementing ICommandSource to avoid unrelated public API members.

Reply:
Because of this:

![image](https://user-images.githubusercontent.com/17993847/151406579-5885869f-67a1-4450-a324-8be4a6d52dda.png)

The thought was why have a public implementation of the interface that just forwards to an internal method with the exact same signature.

Reply:
It's not a public implementation of the interface.
Interface is public, but implementation is **implicit** for a reason to avoid unrelated public API members on Button type.

Reply:
I'm not sure I understand. If the Button implements the ICommandSource interface, which is public, and all members have to be public (at least before new versions of C#) that method will be implicitly public -- which is still public. Code is going to function exactly the same as external code is still going to see a public CanExecuteChanged.

Is this for auto generating docs or something?

Reply:
I tested and you are right. I don't know how something works with interfaces here. Will change back.